### ISSUE TITLE
mate: rendering issue in mate-display-properties

### ISSUE THREAD

**Issue by oblongau** [linuxmint/mint22-beta#186]
This has been a longstanding problem in earlier versions (MATE) on different hardware and is still in MATE 22 beta live CD.

Connect a second (or more) monitor screen to the computer. System correctly recognises the extra screen.

Go to Display > Monitor Preferences and drag a screen to a different location (i.e, to move, say, from left to right). That preview panel does not cleanly show the change, leaving trails of the movement. 

(The example is for three screens, trying to move the right hand screen.)

The corruption remains until that window is closed then re-opened. The screen re-location does actually work.

![display-monitor-preferences](https://github.com/user-attachments/assets/fb7041e9-cb95-42e8-a37d-2e3c01b219c7)

**Comment by clefebvre** [comment]
Know issue upstream: https://github.com/mate-desktop/mate-control-center/issues/713

raveit indicates this can be fixed by modifying the theme, can you try and see if that works?

**Comment by oblongau** [comment]
Simply changing themes made no difference. 

On a freshly installed 21.3 MATE in ~/.config/gtk-3.0 I created gtk.css and pasted in the following as per the link above:

`foo-scroll-area:selected,
foo-scroll-area:selected:focus,
foo-scroll-area:selected:backdrop {
	background-color: @theme_selected_bg_color;
}`

Monitor Preferences when re-arranging monitors was then clean (I did not have to change or refresh the theme).
This also worked on another 21.3 machine that had an existing gtk.css file.

For MATE 22 beta live CD I had to create the gtk-3.0 folder first, then the gtk.css file. That then worked, too.

I would assume some fix is needed so that this does not have to be done on an individual user basis.
